src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,27 @@
 package mate.academy;
 
-import mate.academy.model.Movie;
+import java.time.LocalDate;
+import mate.academy.lib.Injector;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
```

Comment:
1. You should test all methods here, not only 'get' and 'findAvailableSessions'
![image](https://user-images.githubusercontent.com/117547096/214906399-da25d78d-7a98-47d5-af8e-23e8dcfff995.png)
2. Please avoid hardcoded ids like '4L', 5L'. Use ids from created objects.